doc/lua_api.txt

```diff
@@ -2281,6 +2360,41 @@ Elements
 **Note**: do _not_ use a element name starting with `key_`; those names are
 reserved to pass key press events to formspec!
 
+Migrating to Real Coordinates
+-----------------------------
+
+In the old system, positions included padding and spacing. Padding is a gap between
+the formspec window edges and content, and spacing is the gaps between items. For
+example, two `1x1` elements at `0,0` and `1,1` would have a spacing of `5/4` between them,
+and a padding of `3/8` from the formspec edge. It may be easiest to recreate old layouts
+in the new coordinate system from scratch.
+
+To recreate an old layout with padding, you'll need to pass the positions and sizes
+through the following formula to re-introduce padding:
+
+```
+pos = (oldpos + 1)*spacing + padding
+where
+    padding = 3/8
+    spacing = 5/4
+```
+
+You'll need to change the `size[]` tag like this:
+
+```
+size = (oldsize-1)*spacing + padding*2 + 1
+```
+
+A few elements had random offsets in the old system. Here is a table which shows these
+offsets when migrating:
+
+| Element |  Position  |  Size   | Notes
+|---------|------------|---------|-------
+| box     | +0.3, +0.1 | 0, -0.4 |
+| button  |            |         | Buttons now support height, so you will need to set h=1 and reposition if h!=1 before
```

Comment:
Buttons now support height, so you'll need to set h=0.5, and also reposition if h!=1 before

Reply:
To keep the line length limit: How about putting a note `*²` there and `2) Buttons now support height ...` below the table?

Reply:
Shouldn't it be like this?
> Buttons now support height, so you'll need to set `h = 2 * 15/13 * 0.35`, and also reposition if `h ~= 15/13 * 0.35` before

(Actually repositioning has to be done anyway.)

Reply:
I'm not sure what I'm supposed to put here:  DS-Minetest's or Rubenwardy's?  I think DS-Minetest's is the accurate one.

Reply:
DS-Minetest's is better.

However, I do think that 0.5 looks better than 0.8076923076923076 as a button height :D